### DISCUSSION TITLE
Dialog Plugin - change 'ok' btn style when enlabled (isValid)

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#8874] [General - Components / Directives / etc]
Hello, I am using a dialog plugin, and I would like to change the 'ok' button color when it is enabled (prop: prompt.isValid). I can change the color (this.$q.dialog({ok: { color: 'green' } })) but I do not know how to vinculate this change with prompt: { isValid: val => myFunction(val) }... like in regular componen color="get-if-is-enabled ? 'green' : 'red'"...

Is there any boiled or easy way of doing this with the dialog plugin (1.- Get the enabled/disabled state of the 'ok' button and 2. Vinculate it with the 'ok' button style (color)).

Thanks for Quasar!

**Comment by ontwikkelfabriek** [comment]
@eosca 

If you make a codepen we can offer better support for you.

**Comment by hawkeye64** [comment]
@eosca I believe what you want to do has to be with dialog plugin wrapping a custom component: https://next.quasar.dev/quasar-plugins/dialog#invoking-custom-component
